streams/src/main/java/org/apache/kafka/streams/processor/internals/DefaultStateUpdater.java

```diff
@@ -885,7 +885,8 @@ public Set<StreamTask> drainRestoredActiveTasks(final Duration timeout) {
                 restoredActiveTasksLock.lock();
                 try {
                     while (restoredActiveTasks.isEmpty() && now <= deadline) {
-                        final boolean elapsed = restoredActiveTasksCondition.await(deadline - now, TimeUnit.MILLISECONDS);
+                        @SuppressWarnings("UnusedLocalVariable")
+                        final boolean ignored = restoredActiveTasksCondition.await(deadline - now, TimeUnit.MILLISECONDS);
```

Comment:
@cadonna do we need to check the result of the await here?

Reply:
It really does not matter in that case whether we got interrupted or the timeout expired. We could consider just removing the local here. It will be flagged by intellij though.

Reply:
Yeah, @lucasbru got it right. Avoiding the flag by IntelliJ was the reason I added that variable.

Reply:
I had to revert this. Spotbugs actually requires handling the return value here.

Reply:
Should we add a comment why it is ok to ignore the result?